docs/source/extending/models.rst

```diff
@@ -1,5 +1,10 @@
-Extending the Interaction Models (Old-Style)
-============================================
+Obsolete Tutorial: Extending the Interaction Models (Old-Style)
+===============================================================
+.. warning::
+
+    This tutorial is no longer valid since we removed the ``EntityRelationEmbeddingModel`` class.
+    in PyKEEN 1.8.2. Use the ``ERModel`` class for custom models now.
```

Comment:
should we remove that part then?

Reply:
Some of the ideas are good and we should port over to new ERmodels but rewriting docs is no fun and I had no motivation this morning (or now)

Reply:
Done in 764c0a56